tests/reporter/kubernetes.go

```diff
@@ -87,12 +87,20 @@ func (r *KubernetesReporter) SpecDidComplete(specSummary *types.SpecSummary) {
 	if r.artifactsDir == "" {
 		return
 	}
-	r.Dump(specSummary.RunTime, false)
+	r.DumpTestNamespaces(specSummary.RunTime)
 }
 
-// Dump dumps the current state of the cluster. The relevant logs are collected starting
-// from the since parameter.
-func (r *KubernetesReporter) Dump(duration time.Duration, isExternal bool) {
+func (r *KubernetesReporter) DumpTestNamespaces(duration time.Duration) {
+	r.dumpNamespaces(duration, tests.TestNamespaces)
+}
+
+func (r *KubernetesReporter) DumpAllNamespaces(duration time.Duration) {
+	r.dumpNamespaces(duration, []string{v1.NamespaceAll})
+}
+
+// Dump dumps the current state of the cluster. The relevant logs from the
+// specified duration are collected, limited to the specified vmiNamespaces.
```

Comment:
nit: Documentation is done on the exposed functions usually.
And the convention is for the name of the method/function to be the first in the text.

Reply:
I'd very happily drop the doc of the private function. But do I *have* to document the new functions I define in this PR? To me, `DumpAllNamespaces` is self explanatory; docs have a tendency to rot and confuse (as the first commit in this PR shows).

Reply:
I personally do not mind, and this is not a formal library API which someone is suppose to consume (as a library).
So I am fine with removing it.